frontend/utility/platform-x11.cpp

```diff
@@ -220,7 +222,7 @@ bool GetDataFilePath(const char *data, string &output)
 
 string GetDefaultVideoSavePath()
 {
-	return string(getenv("HOME"));
+	return string(g_get_user_special_dir(G_USER_DIRECTORY_VIDEOS));
```

Comment:
Are there scenarios where this might return a `nullptr`? Because the `std::string` will throw an exception if passed such a pointer.

Reply:
Add `getenv("HOME")` if Glib somehow fails to return a path.